src/mechanics/cell/CellData.h

```diff
@@ -14,11 +14,23 @@ namespace NuTo
 class CellData
 {
 public:
-    CellData(const ElementCollection& elements)
+    CellData(const ElementCollection& elements, int numIntegrationPoints, int cellID)
```

Comment:
Why `numIntegrationPoints`? Maybe because you chose to initialize the history data in your test via the cell interface. This is not the right way, IMO. 
Even if we choose to allow a varying number of integration points, this should not be here. We _know_ that in our main file and can initialize our history data there.

Reply:
I ll try an alternative in the next push

Reply:
Okay, this cellGroup-Integrand container might do the trick with the initialization, but since it is more like an own topic, should we for now stick to the manual initialization and postpone the automated initialization if we have this kind of class? I mean, I have a clear picture how it would work for a standard history data container. However, I am not sure if we would add constraints to the custom versions and I think we don't want to add constraints because of default solutions.

Reply:
Removed the numIntegration points. The value is (temporarily) passed directly to Initialize history data